sqlalchemy_solr/solrdbapi/_solrdbapi.py

```diff
@@ -345,19 +345,19 @@ def connect(
     server_path="solr",
     collection=None,
     use_ssl=False,
-    verify_ssl=False,
+    verify_ssl=None,
     ca_certs=None,
     token=None,
 ):
 
     session = Session()
     mf = MessageFormatter()
 
-    if verify_ssl is False:
+    if verify_ssl is not None and verify_ssl in [False,"False","false"]:
         session.verify = False
     else:
         if ca_certs is not None:
-            session.verify = ca_certs
+            session.cert = ca_certs
```

Comment:
Hi @rohitpawar2811 I suggest keeping `verify_ssl` in parameters and delegating CA bundle specification to `REQUESTS_CA_BUNDLE` environment variable, i.e. removing `ca_certs` from the parameters and using the environment variable instead.

Actually, `session.verify` is the correct attribute but I couldn't get it working despite `curl` works fine using the same inputs. Were you able to get `session.verify` working at your end? If yes, then `REQUESTS_CA_BUNDLE` should do as well.

Reply:
>> Were you able to get session.verify working at your end
Yes its working on my end

Reply:
1. By using REQUESTS_CA_BUNDLE, we don't have to include the certificates in each and every request. I think that was your intent behind this, or is there something else? Just want to confirm.

2. Are we still expecting the 'ca_certs' path to come from Superset, or not? 
os.environ['REQUESTS_CA_BUNDLE'] = ca_certs. 

However, a situation come : if I have connected to solr1 with cert1, and now I am calling any request to solr2 with cert2, an error might occurs,as REQUESTS_CA_BUNDLE certificate has changed.

Reply:
1. Yes, and that applies to database connections as well.
2. If `REQUEST_CA_BUNDLE` is used, Requests library handles it automatically and it's not expected to be passed by Superset.
3. In that situation, a CA bundle is created that combines cert1 and cert2 (and any additional number of certificates), therefore all Solr connections work fine without changing `REQUESTS_CA_BUNDLE`.

Reply:
1. Okay then according to me now I have to remove the code of certs and rest of things as is.
2. As its not expected to pass from superset, 
so anyone can declare the env and set path/to/cacert.

Reply:
Yes, `ca_certs` related code should be removed and `REQUESTS_CA_BUNDLE` can be used instead. The remining are as is.

Reply:
please check I have did the changes